instrumentation/pulsar/pulsar-2.8/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/pulsar/v2_8/AbstractPulsarClientTest.java

```diff
@@ -187,7 +209,25 @@ void testConsumeNonPartitionedTopicUsingBatchReceive() throws Exception {
                                             equalTo(MESSAGING_DESTINATION_NAME, topic),
                                             equalTo(SERVER_PORT, brokerPort),
                                             equalTo(SERVER_ADDRESS, brokerHost))
-                                        .hasBucketBoundaries(DURATION_BUCKETS))));
+                                        .hasBucketBoundaries(DURATION_BUCKETS))),
+            metric ->
+                OpenTelemetryAssertions.assertThat(metric)
+                    .hasName("messaging.receive.messages")
+                    .hasUnit("{message}")
+                    .hasDescription("Measures the number of received messages.")
+                    .hasLongSumSatisfying(
+                        sum -> {
+                          sum.hasPointsSatisfying(
+                              point -> {
+                                point
+                                    .hasValue(batchSize.get())
```

Comment:
isn't this always `1`, do you even need to record the batch size

Reply:
Done